joblib/memory.py

```diff
@@ -470,6 +470,35 @@ def __init__(self, func, cachedir, ignore=None, mmap_mode=None,
             doc = func.__doc__
         self.__doc__ = 'Memoized version of %s' % doc
 
+    def is_cached(self, *args, **kwargs):
+        """Check to see if a call to the wrapped function with these arguments
+        would result in a cache hit.
+
+        Returns
+        -------
+        cached: bool
+            Whether or not the call is already cached
+        """
+        output_dir, argument_hash = self._get_output_dir(*args, **kwargs)
+        return self._needs_call(output_dir, argument_hash)
+
+    def _needs_call(self, output_dir, argument_hash):
```

Comment:
`argument_hash` is not used by the implementation of this function. I think it can be removed.